contracts/common/ERC20Upgradeable.sol

```diff
@@ -0,0 +1,425 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I assume here you are extending/overwriting OZ's `ERC20Upgradeable.sol`. Can you inherit from it so we only check in the changes?

If that's not possible for some reason adding a comment stating what is the change that you introduced, hard to tell from this file.

Reply:
I added a comments since only change is to have _balances internal instead of private which cannot be solved by inheritance